src/components/Person/Person.jsx

```diff
@@ -1 +1,19 @@
-// export const Person = ({ person }) => ();
+import React from 'react';
+
+export const Person = ({ person }) => {
+  const personPartnerParagraph = person.isMarried ? (
+    <p className="Person__partner">
+      {`${person.partnerName} is my ${person.sex === 'm' ? 'wife' : 'husband'}`}
+    </p>
+  ) : (
+    <p className="Person__partner">I am not married</p>
```

Comment:
You can also you only one tag `<p>` and use ternary only for content

Reply:
Done